source/about/embed-mattermost-within-microsoft-teams.rst

```diff
@@ -1,20 +1,15 @@
-Install the Mattermost App in Microsoft Teams
-=============================================
-
+Embed Mattermost within Microsoft Teams
+=======================================
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-.. tip::
-
-  Looking to `install the Mattermost for Microsoft Teams plugin </about/install-mattermost-for-microsoft-teams-plugin.html>`__ instead?
-
-With `Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__, you can embed your Mattermost workspace within your Microsoft Teams instance and take advantage of `embedded app features <#benefits-of-the-embedded-app>`_.
+With the `Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__ integration, you can embed your Mattermost workspace within your Microsoft Teams instance and take advantage of `embedded app features <about/mattermost-for-microsoft-teams.html#benefits-of-embedding-mattermost>`__.
 
 .. important::
 
-    You must `install the Microsoft Teams plugin in Mattermost </plugins/install-mattermost-for-microsoft-teams-plugin>`__ before you can embed the Mattermost app within Microsoft Teams.
+    You must `install the Microsoft Teams plugin in Mattermost </about/install-mattermost-for-microsoft-teams-plugin.html>`__ before you can embed the Mattermost app within Microsoft Teams.
```

Comment:
How can we differentiate this with the "Microsoft Teams Connector plugin"?

Reply:
It's my understanding that admins need to install the MS Teams integration in order to be able to embed Mattermost in an MS Teams instance. Is that understanding correct?

Reply:
Correct. See latest note for more info